inspirehep/modules/literaturesuggest/tasks.py

```diff
@@ -208,14 +209,17 @@ def formdata_to_model(obj, formdata):
     # ======================================
     # Journal name Knowledge Base conversion
     # ======================================
-    if data.get("publication_info", [{}])[0].get("journal_title"):
-        # journals_kb = dict([(x['key'].lower(), x['value'])
-        #                     for x in get_kb_mappings(current_app.config.get("DEPOSIT_INSPIRE_JOURNALS_KB"))])
-
-        # data['publication_info']['journal_title'] = journals_kb.get(data['publication_info']['journal_title'].lower(),
-        #                                                                 data['publication_info']['journal_title'])
-        # TODO convert using journal records
-        pass
+    journal_title = get_value(data, "publication_info[0].journal_title")
+    if journal_title:
+        title_translation = JournalsSearch().query(
+            'match', title_variants__title__raw=journal_title
+        ).execute()
+        if title_translation:
+            try:
+                data["publication_info"][0]["journal_title"] = \
+                    title_translation[0].short_titles[0].title
+            except (AttributeError, IndexError):
+                pass
```

Comment:
Needs tests. I'd mock `execute` and write unit tests for this alongside the others in https://github.com/inspirehep/inspire-next/blob/b7f8c916ae3786d26533dd23c2868e0bd6244a11/tests/unit/literaturesuggest/test_literaturesuggest_tasks.py.